test/playback.test.js

```diff
@@ -28,6 +28,13 @@ const playFor = function(player, time, cb) {
   checkPlayerTime();
 };
 
+let testFn = 'test';
+
+// TODO: get these tests working, right now we just one the one basic test
```

Comment:
These tests fail sometimes on ie 11 and edge, if we can find out how to fix that we can turn them back on.